acceptance-tests/support/steps/input-steps.ts

```diff
@@ -0,0 +1,40 @@
+import {When, Then} from "@cucumber/cucumber";
+import {enterTextIntoTextInput, checkErrorMessageDisplayedForField, clickSubmitButton} from "./shared-functions";
+
+const fields: Record<string, string> = {
+    name: "userName",
+    email: "emailAddress",
+    password: "password",
+    "security code": "securityCode",
+    "department name": "department",
+    "mobile phone number": "mobileNumber"
+};
+
+When("they submit the {} {string}", async function (fieldName, value) {
+    await enterTextIntoTextInput(this.page, value, fields[fieldName]);
+    await clickSubmitButton(this.page);
+});
+
+When("they enter the {} {string}", async function (fieldName, value) {
+    await enterTextIntoTextInput(this.page, value, fields[fieldName]);
+});
+
+When("the user submits the security code", async function () {
+    await enterTextIntoTextInput(this.page, "123456", "securityCode");
```

Comment:
```suggestion
    await enterTextIntoTextInput(this.page, "123456", fields["security code"]);
```
Let's keep the field names in one place and refer to them so it's easier to change and manage